### PR TITLE
Add support for GL_EXT_YUV_target

### PR DESCRIPTION
**Pull request by nicebyte**
This PR adds support for the [GL_EXT_YUV_target](https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_YUV_target.txt) extension.

### CODE REVIEW

File: glslang/MachineIndependent/Versions.cpp [hunk 1]
```diff
@@ -157,6 +157,7 @@ void TParseVersions::initializeExtensionBehavior()
     extensionBehavior[E_GL_EXT_frag_depth]                   = EBhDisable;
     extensionBehavior[E_GL_OES_EGL_image_external]           = EBhDisable;
     extensionBehavior[E_GL_OES_EGL_image_external_essl3]     = EBhDisable;
+    extensionBehavior[E_GL_EXT_YUV_target]                   = EBhDisable;
```
[the remark this answers is not in the document]
Comment (nicebyte) [reply glslang/MachineIndependent/Versions.cpp]: Added

File: glslang/MachineIndependent/Versions.cpp [hunk 2]
```diff
@@ -307,6 +308,7 @@ void TParseVersions::getPreamble(std::string& preamble)
             "#define GL_EXT_frag_depth 1\n"
             "#define GL_OES_EGL_image_external 1\n"
             "#define GL_OES_EGL_image_external_essl3 1\n"
+            "#define GL_EXT_YUV_target\n"
```
Comment (johnkslang) [inline glslang/MachineIndependent/Versions.cpp]: The `1` is missing? Is that expected?

Maybe this is missing in the extension specification, and should be added to it, probably in the way all the others are done, unless the extension specification somehow does say it's just a simple `#define`.

File: glslang/Include/Types.h [hunk 3]
```diff
@@ -233,6 +236,9 @@ struct TSampler {   // misnomer now; includes images, textures without sampler,
             s.append("ExternalOES");
             return s;
         }
+        if (yuv) {
+          return "__" + s + "External2DY2YEXT";
```
Comment (johnkslang) [inline glslang/Include/Types.h]: 4 spaces (fix if the other comment below needs a fix).

File: glslang/MachineIndependent/Versions.cpp (hunk 2 above)
Comment (nicebyte) [reply glslang/MachineIndependent/Versions.cpp]: nope, i just missed it. fixed.

**Pull request merged**